splice_access_api/src/main/java/com/splicemachine/access/configuration/HBaseConfiguration.java

```diff
@@ -108,7 +108,7 @@
      * Defaults to 8192
      */
     public static final String TIMESTAMP_BLOCK_SIZE = "splice.timestamp_server.blocksize";
-    protected static final int DEFAULT_TIMESTAMP_BLOCK_SIZE = 8192;
+    protected static final int DEFAULT_TIMESTAMP_BLOCK_SIZE = 32768;
```

Comment:
Please change the comment to reflect the default TS block size change from 8k to 32k. But why do you propose the change?